fe/fe-core/src/main/java/com/starrocks/analysis/ColumnDef.java

```diff
@@ -95,13 +108,18 @@ public DefaultValueDef(boolean isSet, Expr expr) {
     private final String comment;
 
     public ColumnDef(String name, TypeDef typeDef) {
-        this(name, typeDef, false, null, false, DefaultValueDef.NOT_SET, "");
+        this(name, typeDef, null, false, null, false, DefaultValueDef.NOT_SET, "");
     }
 
-    public ColumnDef(String name, TypeDef typeDef, boolean isKey, AggregateType aggregateType,
+    public ColumnDef(String name, TypeDef typeDef, String charsetName, boolean isKey, AggregateType aggregateType,
                      Boolean isAllowNull, DefaultValueDef defaultValueDef, String comment) {
         this.name = name;
         this.typeDef = typeDef;
+        if (charsetName == null) {
```

Comment:
add new constructor for String charsetName
and set charsetName to utf8 without set to null may better